src/main/webapp/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution"/>
+      <button onclick="loadSchoolInfo()">Search</button>
+    </header>
+    <div id="information">
+      <h2>Our Focus</h2>
+      <p>Black Student Insight serves as a platform for current Black college students
+        to share their experiences and beliefs regarding their campuses, while also
+        providing prospective students with general information about the schools,
+        such as overall student diversity demographics, admissions statistics, and
+        tuition for their university of interest.
+      </p>
+
+      <h2>Meet The Creators</h2>
+      <div id="headshots">
+        <figure class="inline-block">
```

Comment:
This topic is a bit controversial, but I'd suggest that you don't name this class "inline-block". I'd prefer applying to all `figure` tags within the element with ID "headshots":

#headshots > figure

Some popular CSS naming conventions would prefer you to do this, but I'm not really sold into complicating the HTML with more classes when you already have a good single CSS selector that can target the right thing.